sdc/tests/test_series.py

```diff
@@ -3168,6 +3168,38 @@ def test_series_str2str(self):
                 pd.testing.assert_series_equal(hpat_func(S), test_impl(S),
                                                check_names=check_names)
 
+    def test_series_capitalize_str(self):
+        def test_impl(S):
+            return S.str.capitalize()
+
+        sdc_func = self.jit(test_impl)
+        s = pd.Series(['lower', 'CAPITALS', 'this is a sentence', 'SwApCaSe'])
```

Comment:
I propose to use `test_global_input_data_unicode_kind4` and `test_global_input_data_unicode_kind1` as input data for such tests.